EIPS/eip-5190.md

```diff
@@ -0,0 +1,59 @@
+---
+eip: 5190
+title: Account-bound Finance
+description: A smart contract design pattern in decentralized finance to remove insolvency and arbitrary loss of funds using blockchain technology only. Extension of EIP-4973
+author: Hyungsuk Kang (@hskang9)
+discussions-to: https://ethereum-magicians.org/t/
+type: Standards Track
+category: ERC
+status: Draft
+requires: 721, 1155, 4973
+created: 2022-06-29
+---
+
+## Abstract
+This EIP proposes a form of smart contract design pattern and a new type account abstraction on how one's finance should be managed in web3.0, ensuring transparency of managing investments and protection with self-sovereignty even from its financial operators.
+
+This EIP also extends account-bound token from [EIP-4973](https://eips.ethereum.org/EIPS/eip-4973), but the difference is that it can be transferred to other account for mobility between managing device.
+
+## Motivation
+
+Finance in crypto ecosystem are facing huge trust issue. Smart contracts can be now a proxy of multisig for operating teams, and Insolvencies do exist in that there is no trace of funds written in blockchain. That can create arbitrary loss of funds as happened in official bankruptcy announcement of Celsius and UST depegging and anchor protocol failure. Smart contract should be used for peer-to-peer only, with immutable rules etched so that only one who has the private key has full access of managing his investment as bitcoin promises. 
```

Comment:
I really can't tell what you're trying to say here. Could you fix the grammar?

Reply:
It is true that there are some jargons used such as depegging, and insolvencies regarding decentralized finance systems that newcomers cannot understand. Can you clarify what is not understandable from your point of view?

Reply:
Based on your points, I will add glossaries and more descriptions to the problem.

Reply:
I did mean grammar, as it turns out. The sentences are ungrammatical to the point that I cannot actually make sense of what you're trying to say.

Reply:
I agree with Pandapip here that as currently written, this paragraph is unintelligible.  While we do our best to help authors who aren't native English speakers clearly express themselves in English, we do need a base to start from that is "good enough" for us to effectively iterate on it.  Unfortunately, this paragraph doesn't meet that bar and so you'll need to figure out a way to get it *good enough* that we can help you improve it further.

My recommendation is to perhaps find a peer who speaks your native language and English and the two of you can iterate on this paragraph together to make it more understandable.